inc/utils.php

```diff
@@ -81,3 +81,17 @@ function cardToken()
 {
     return $cardToken = filter_input(INPUT_POST, 'cardToken', FILTER_SANITIZE_STRING) ?: '';
 }
+
+/**
+ * Retrieve the Payment Methods from the transient or the Api
+ *
+ * @return array|bool|mixed|\Mollie\Api\Resources\BaseCollection|\Mollie\Api\Resources\MethodCollection
+ */
+function getApiMethods()
```

Comment:
I don't like to use `get` on methods or functions but a part that the documentation is saying _Retrieve the Payment Methods from the transient or the Api_ but the name of the function is `getApiMethods` would be better to express better what the function does/return. 

if it's called `getApiMethods` I could think I would receive a list of the Methods allowed for Api requests, such as PUT, HEAD etc... where instead I'm receiving a list of allowed payment methods.

I would suggest then to rename it to `availablePaymentMethods`

Also the documentation can be improved because the consumer is not interested to know what a transient is and if caching exists if not explicly specified.